src/Images.jsx

```diff
@@ -35,6 +35,7 @@ class Images extends React.Component {
         this.state = {
             intermediateOpened: false,
             isExpanded: false,
+            imageDownloadInProgress: [],
```

Comment:
This could use a comment what the structure of that state is -- it's not obvious from an empty list. I think it's a list of strings with image names?